projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewService.java

```diff
@@ -175,23 +175,23 @@ public List<ViewDTO> getViews() {
         return viewsConfig.views.stream()
                 .map(v -> {
                     var columns = new ArrayList<ColumnDTO>();
-                    columns.add(new ColumnDTO(v.name, v.itemName == null ? v.name : v.itemName, ColumnType.Identifier));
+                    columns.add(new ColumnDTO(v.name, v.itemName == null ? v.name : v.itemName, ColumnType.Identifier, 0));
```

Comment:
could you please make the 0 as a const with proper naming for better readability?

Reply:
I think this is a bit to much, since the variable isn't used elsewhere. Also there is the IDE assistance:

![image](https://github.com/thehyve/fairspace/assets/29271979/6816b87b-dbbc-4bad-b5a1-1d7dc15cffb3)